### ISSUE TITLE
DB credentials checked into codebase

### ISSUE THREAD

**Issue by harlantwood** [willzeng/cograph#643]
Recommend fix: use the awesome [dotenv](https://www.npmjs.com/package/dotenv) package to load environment variables either from process.env (if they exist there, as on Heroku) or from a local .env file, typically on a dev box.

I will create a PR once #642 is merged (otherwise the diff will not make sense) -- I created a PR in my own repo to show a demo of how this would work: https://github.com/harlantwood/cograph/pull/1/files

**Comment by harlantwood** [comment]
Also we would want notes in the README: 

> To get started in your local dev env:
> `cp .env.example .env` 
> Change any credentials to your own databases, etc

**Comment by willzeng** [comment]
Yep we should go in this direction as soon as we sort the CI in #642. +1
